.github/workflows/tests.yml

```diff
@@ -0,0 +1,54 @@
+name: Tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest]
+        php: [8.2, 8.1]
+        laravel: [9.*]
```

Comment:
Is it possible to already test with 10.*?

Reply:
✅